examples/organization/README.md

```diff
@@ -88,7 +88,8 @@ module "secure-for-cloud_example_organization" {
   }
 
   source = "sysdiglabs/secure-for-cloud/google//examples/organization"
-  organization_domain       = "<ORG_DOMAIN>"
+  organization_domain = "<ORG_DOMAIN>"
+  name                = "<NAME>" 
```

Comment:
in docs we only put mandatory attributes, or if optional, state it's optional to avoid confussion

Reply:
Removed this.